src/components/CommentList.jsx

```diff
@@ -0,0 +1,57 @@
+import {useEffect, useState} from 'react';
+import {fetchCommentsByArticleId, postCommentByArticleId} from '../api';
+import { useParams } from 'react-router-dom';
+import Comment from './Comment';
+
+const CommentList = ({username}) => {
+    const [comments, setComments] = useState([]);
+    const [commentText, setCommentText] = useState("");
+    const {article_id} = useParams();
+
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        console.log(commentText);
+        console.log(username);
+        const comment = {
+            username: username,
+            body: commentText
+        };
+        postCommentByArticleId(article_id, comment).then((data) => {
+            setComments((currComments) => {
+                return [...currComments, data.comment];
+            });
+        });
+        setCommentText("");
+    }
+
+    useEffect(() => {
+        fetchCommentsByArticleId(article_id).then((data) => {
+            console.log("I am Looping");
```

Comment:
logs! 😂

Reply:
that one is because I am scared to break something